kong/llm/drivers/anthropic.lua

```diff
@@ -68,8 +68,22 @@ local transformers_to = {
       return nil, nil, err
     end
     
-    prompt.temperature = (model.options and model.options.temperature) or nil
-    prompt.max_tokens_to_sample = (model.options and model.options.max_tokens) or nil
+    prompt.temperature = request_table.temperature
+                      or (model.options and model.options.temperature) 
```

Comment:
Do we control the full lifecycle of this `model` table? If so, maybe we can initialize `model.options` to an empty table at some point early on such that these expressions can be simplified.

Reply:
I didn't want to do this, in case I override any functionality that comes with the plugin_conf table that is passed into the handler.

Feels dangerous, and better to just re-create each field.